app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicListFragment.kt

```diff
@@ -12,6 +12,7 @@ import javax.inject.Inject
 class OngoingTopicListFragment : InjectableFragment() {
 
   companion object {
+    internal const val TAG_ONGOING_TOPIC_LIST_FRAGMENT = "TAG_ONGOING_TOPIC_LIST_FRAGMENT"
```

Comment:
Rename to `ONGOING_TOPIC_LIST_FRAGMENT_TAG`

Reply:
done